.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,18 @@
+*   Version of collectd:
+*   Operating system / distribution:
+
+## Expected behavior
+
+(Description of the behavior / output that you expected)
+
+## Actual behavior
+
+(Description of the behavior / output that you observed)
+
+## Steps to reproduce
+
+*   step 1
+*   step 2
+*   step 3
+
+<!-- vim: set syntax=markdown : -->
```

Comment:
Nice, and simple enough so people might actually fill it in ;)

One suggestion, could you name this ISSUE_TEMPLATE.md?